kubernetes-setup/main.yaml

```diff
@@ -52,9 +52,14 @@
     - name: Disable swap in fstab
       replace:
         path: /etc/fstab
-        regexp: '^([^#].*\s+swap\s+swap\s+.*)'
+        regexp: '^([^#].*\s+swap\s+.*)'
         replace: '# \1'
 
+    - name: Check kvm module
+      modprobe:
+         name: kvm
+         state: present
+
```

Comment:
May be we can use this with  `SetupKataContainers` condition since this will specifically apply for Kata. 
What do you say ?

Reply:
Sure, changed, would it make sense as it now?

Reply:
Yep.. Thanks @snir911